web3/utils/compat/compat_codecs.py

```diff
@@ -0,0 +1,24 @@
+
+import codecs
+import sys
+
+
+def backslashreplace(ex):
+    # The error handler receives the UnicodeDecodeError, which contains arguments of the
+    # string and start/end indexes of the bad portion.
+    bstr, start, end = ex.args[1:4]
+
+    # The return value is a tuple of Unicode string and the index to continue conversion.
+    # Note: iterating byte strings returns int on 3.x but str on 2.x
+    return (
+        u''.join(
```

Comment:
You should be able to remove the `u` by adding a `from __future__ import unicode_literals` to the top of this module.